x-pack/plugins/maps/public/connected_components/toolbar_overlay/tools_control/tools_control.test.tsx

```diff
@@ -30,7 +32,7 @@ test('renders', async () => {
 });
 
 test('Should render cancel button when drawing', async () => {
-  const component = shallow(<ToolsControl {...defaultProps} isDrawingFilter />);
+  const component = shallow(<ToolsControl {...defaultProps} />);
```

Comment:
I think this test case should set filterModeActive to true to capture the test case of showing the cancel button. If you look in the snapshot, the cancel button is not there for this test case.